source/aggregation-pipeline-builder.txt

```diff
@@ -89,69 +89,225 @@ Limitations
+               :alt: Toggle on stage view mode 
+
+         .. step:: At the bottom of the aggregation pipeline pane, select the 
+            :guilabel:`+ Add Stage` button to open a new aggregation stage 
+            modal.
+            
+         .. step:: At the lower-left corner of the aggregation pipeline pane, 
+            click the :guilabel:`Select...` dropdown and select the 
+            :manual:`aggregation pipeline stage 
+            </reference/operator/aggregation-pipeline/>` to use for the first 
+            stage of the pipeline. 
+
+         .. step:: Fill in your selected pipeline stage. 
+            
+            If you enable :guilabel:`Preview`, preview documents update 
+            automatically to the right of the stage to reflect the results of 
+            your pipeline as it progresses. You can adjust the width of the 
+            pipeline stage by dragging its border to the right.
+            
+            .. figure:: /images/compass/agg-builder-match-stage.png 
+               :alt: Aggregation Builder match stage example
+         
+         .. step:: Click :guilabel:`Add Stage` to add additional aggregation 
+            stages below your most recently added aggregation stage. Repeat 
+            steps 1 and 2 for each additional stage.
+
+            .. note::
+               
+               .. include:: /includes/fact-agg-pipeline-preview-toggle.rst 
+               
+               For example, the following pipeline excludes the first 
+               :manual:`$match </reference/operator/aggregation/match/>` stage 
+               and only includes the 
+               :manual:`$project </reference/operator/aggregation/project/>`
+               stage:
+               
+               .. figure:: /images/compass/agg-builder-exclude-stage-example.png 
+                  :alt: Aggregation Builder exclude stage example
+
+         .. step:: Click :guilabel:`Run` at the top right of the pipeline 
+            builder. |compass-short| returns your results in the document view. 
+            
+            .. warning:: 
+               
+               When you click :guilabel:`Run`, the aggregation runs against the 
+               entire collection.
+
+         .. step:: To edit the pipeline, click :guilabel:`Edit` at the top of 
+            your pipeline toolbar to return to the aggregation pipeline builder
+            and make changes to your pipeline. 
+
+   .. tab:: Focus Mode 
+      :tabid: focus 
+
+      .. procedure::
+         :style: normal
+         
+         .. step:: Select the :guilabel:`Stages` view.
+            
+            In the aggregation pipeline pane, ensure the 
+            :guilabel:`{} Stages` toggle switch is selected.
+
+            .. figure:: /images/compass/agg-builder-stageview-toggle-on.png
+               :alt: Toggle on stage view mode 
+
+         .. step:: Open the aggregation stage modal.
```

Comment:
The second step is to add a stage if there's none. User can simply click on Add Stage button and it will create a stage card, where we have a Focus Mode button. 

The *open* & *modal* in this title confuses me.

Reply:
Noted. Removing language about the "modal" and adding the note about needing to add a stage "if there hasn't already been one added".